src/Codeception/Configuration.php

```diff
@@ -646,7 +646,7 @@ private static function prepareParams($settings)
             }
 
             // .env and ini files
-            if (preg_match('~(\.ini|\.env)$~', $paramStorage)) {
+            if (preg_match('~(\.ini$|\.env)~', $paramStorage)) {
```

Comment:
@stipsan I see two reasons to "limit" .ini files:
1. They were "limited" before, so I don't change anything in that regard.
2. I've never seen a config file named `.ini.something`.

examples of file names that would match:
```
prod.ini
testing.ini
.env
.env.testing
```